src/test/java/com/google/api/codegen/testdata/python_README_library.baseline

```diff
@@ -2,7 +2,9 @@
 Python Client for Google Example Library API (`Alpha`_)
 ==================================================================================================
 
-Idiomatic Python client for `Google Example Library API`_
+Idiomatic Python client for `Google Example Library API`_:
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Ah, so I think with reST the header level is determined implicitly. That means there's no way to avoid having the tilde be interpreted as h2 here (which then throws off the other heading levels).